### ISSUE TITLE
CSP issue

### ISSUE THREAD

**Issue by oliviertassinari** [alitaheri/normalize-scroll-left#1]
> Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: ...

This warning is coming from:

```js
var isBrowser = new Function('try {return this===window;}catch(e){ return false;}');
```

Have you consider an implementation like this one https://github.com/react-bootstrap/dom-helpers/blob/master/src/util/inDOM.js
?

**Comment by alitaheri** [comment]
@oliviertassinari Looks a lot better, 😅 Thanks

**Comment by oliviertassinari** [comment]
We use it here, it's tested and working fine https://github.com/callemall/material-ui/blob/v1-beta/src/internal/Modal.js#L9 :)

**Comment by alitaheri** [comment]
@oliviertassinari It's done. I've published `0.1.1` 😁

**Comment by oliviertassinari** [comment]
@alitaheri Thanks.
